modules/oe_theme_content_news/src/Plugin/PageHeaderMetadata/NewsContentType.php

```diff
@@ -98,9 +100,16 @@ public function getMetadata(): array {
       ];
     }
 
+    $news_type = $node->get('oe_news_types')->entity;
+    if ($news_type instanceof ConceptInterface) {
+      $cacheability = new CacheableMetadata();
+      $cacheability->addCacheableDependency($news_type);
+      $news_type = \Drupal::service('entity.repository')->getTranslationFromContext($news_type);
```

Comment:
This service needs to be injected, and we need to print all values, not only the first one, as the field is multiple. We should also test that, on both teaser and full page.